### DISCUSSION TITLE
Add customized action items.

### DISCUSSION THREAD

**Discussion by emnimata** [avo-hq/avo#604] [Feature requests]
It will be nice if we can customize the action items links and behavior when we define a resource.

class MyResource < Avo::BaseResource
  action_items only: :Index do
     link_to "Custom Action", new_admin_post_path(post)
  end
end

**Comment by adrianthedev** [comment]
I see what you mean. Yeah, that could be a nice way to control the action on a per resource basis. I'll add it on the roadmap after dashboards and mobile interface.

**Comment by adrianthedev** [comment]
This is fixed using [`actions_list`](https://docs.avohq.io/3.0/customizable-controls.html#actions_list)
